client/webpack.client.base.config.js

```diff
@@ -54,12 +55,27 @@ module.exports = {
   ],
   module: {
     loaders: [
+      { test: /\.(woff2?|svg)$/, loader: 'url?limit=10000' },
+      { test: /\.(ttf|eot)$/, loader: 'file' },
+      { test: /\.(jpe?g|png|gif|svg|ico)$/, loader: 'url?limit=10000' },
 
-      // React is necessary for the client rendering:
+      // React is necessary for the client rendering
       { test: require.resolve('react'), loader: 'expose?React' },
       { test: require.resolve('react-dom'), loader: 'expose?ReactDOM' },
       { test: require.resolve('jquery'), loader: 'expose?jQuery' },
       { test: require.resolve('jquery'), loader: 'expose?$' },
+
+      // Use one of these to serve jQuery for Bootstrap scripts:
+
+      // Bootstrap 3
+      { test: /bootstrap-sass\/assets\/javascripts\//, loader: 'imports?jQuery=jquery' },
+
+      // Bootstrap 4
+      { test: /bootstrap\/dist\/js\/umd\//, loader: 'imports?jQuery=jquery' },
     ],
   },
+
+  postcss: [autoprefixer],
```

Comment:
`postcss-loader`, we can apply it and it will apply all postcss plugins placed in this array.